### ISSUE TITLE
Add building frontend for staging and production deployment

### ISSUE THREAD

**Issue by manekenpix** [Seneca-CDOT/telescope#658]
**What would you like to be added**:

Currently, we have scripts in our `package.json` to build the front end, but they have to be run manually. For staging and (maybe) production deployment, building the front end needs to happen automagically, otherwise the backend (or nginx once that part is added) won't have anything to serve.

Our [Docker](https://github.com/Seneca-CDOT/telescope/blob/master/Dockerfile) file (not `docker-compose.yml`) is responsible for setting up the container for the Telescope app. We probably need to add something there to build the front end right after installing all the dependencies.

Something similar to:

```yml
...
COPY package.json ./
COPY .npmrc ./

# Install all Node.js modules on the image
RUN npm install --no-package-lock
RUN npm run build     <-- This would build the frontend before deployment

# Bundle app source
COPY . .
...
```

**Comment by humphd** [comment]
We'd also need to `ADD` or `COPY` the `src/frontend/public` folder after that build step completed, I think.

**Comment by manekenpix** [comment]
Closed via #678
